sdk/tables/azure-data-tables/azure/data/tables/_generated/_configuration.py

```diff
@@ -37,7 +40,7 @@ def __init__(
 
         self.url = url
         self.version = "2019-02-02"
-        kwargs.setdefault('sdk_moniker', 'azuretable/{}'.format(VERSION))
+        kwargs.setdefault('sdk_moniker', 'table/{}'.format(VERSION))
```

Comment:
this too, I think this should be data-tables/


Wondering if we need to regenerate with autorest and specify namespace like, 
--namespace=azure.data.tables --package-name=azure-data-tables

@iscai-msft

Reply:
@kristapratico yeah the correct sdk moniker should be 'data-tables'. But we shouldn't need to regenerate, since we're overwriting the generated code's sdk moniker [here](https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/tables/azure-data-tables/azure/data/tables/_base_client.py#L389). Instead, the code I linked should be changed to "data-tables"

Reply:
unrelated to user-agent, I think we still need to regen with azure-data-tables otherwise all generated code refs will be off

Reply:
yeah ended up agreeing with you after going through the whole SDK. I don't think it's 100% necessary since it's more for us, but do agree it would look a lot better

Reply:
@kristapratico @iscai-msft I regenerated this, user-agent isn't used anywhere in this sdk, should I move it to it's own file or should I remove it entirely?

Reply:
You can move it to its own file like how izzy linked above, it's more manageable that way. It looks like the current user-agent string is

azsdk-python-storage-table/2019-07-07 Python/3.8.3 (Windows-10-10.0.19041-SP0)

and it should look like this:

azsdk-python-**data**-table**s**/**{SDK version}** Python/3.8.3 (Windows-10-10.0.19041-SP0)